### PR TITLE
Add kubeadm to “Install Tools”

### PR DESCRIPTION
**Pull request by JornShen**
ref: https://github.com/kubernetes/website/issues/24532

### CODE REVIEW

**Comment by JornShen** [comment]
How about it? @sftim

File: content/en/docs/tasks/tools/_index.md [hunk 1]
```diff
@@ -47,3 +47,13 @@ installed and configured.
 you need to do to get up and running with `kind`.
 
 <a class="btn btn-primary" href="https://kind.sigs.k8s.io/docs/user/quick-start/" role="button" aria-label="View kind Quick Start Guide">View kind Quick Start Guide</a>
+
+## kubeadm
+
+[`Kubeadm`](https://github.com/kubernetes/kubeadm) is a tool built to provide best-practice "fast paths" for creating Kubernetes clusters.
```
Comment (neolit123) [inline content/en/docs/tasks/tools/_index.md:53]: We should link to the installing kubeadm guide at k8s.io instead.

**Review by neolit123 — COMMENTED** [review]
Thanks for the PR. Added one comment.

/sig cluster-lifecycle

File: content/en/docs/tasks/tools/_index.md [hunk 2]
```diff
@@ -47,3 +47,13 @@ installed and configured.
...
+It performs the actions necessary to get a minimum viable, secure cluster up and running in a user friendly way.
+
+[`Quick Intall`](https://kubernetes.io/docs/setup/production-environment/tools/kubeadm/install-kubeadm/) show you how to install `kubeadm`.
```
Comment (sftim) [inline content/en/docs/tasks/tools/_index.md:56]:
```suggestion
[`Quick Install`](/docs/setup/production-environment/tools/kubeadm/install-kubeadm/) show you how to install `kubeadm`.
```

File: content/en/docs/tasks/tools/_index.md [hunk 3]
```diff
@@ -47,3 +47,13 @@ installed and configured.
...
+And After install, you can use it to [create you clusters](https://kubernetes.io/docs/setup/production-environment/tools/kubeadm/create-cluster-kubeadm/).
```
Comment (sftim) [inline content/en/docs/tasks/tools/_index.md:57]:
```suggestion
Once installed, you can use it to [create a cluster](/docs/setup/production-environment/tools/kubeadm/create-cluster-kubeadm/).
```

**Review by sftim — CHANGES_REQUESTED** [review]
Here are a couple of suggestions.
- correct spellings
- use site-relative hyperlinks

File: content/en/docs/tasks/tools/_index.md (hunk 2 above)
Comment (JornShen) [reply content/en/docs/tasks/tools/_index.md:56]: thx! done

File: content/en/docs/tasks/tools/_index.md (hunk 3 above)
Comment (JornShen) [reply content/en/docs/tasks/tools/_index.md:57]: thx! done

File: content/en/docs/tasks/tools/_index.md (hunk 1 above)
Comment (JornShen) [reply content/en/docs/tasks/tools/_index.md:53]: Do you mean change "https://github.com/kubernetes/kubeadm" to "/docs/setup/production-environment/tools/kubeadm/install-kubeadm/" ? 

If so, I still think here link to "https://github.com/kubernetes/kubeadm" is more suitable. 🤔

Comment (sftim) [inline content/en/docs/tasks/tools/_index.md:53]:
```suggestion
You can use the {{< glossary_tooltip term_id="kubeadm" text="kubeadm" >}} tool to create and manage Kubernetes clusters.
```

Comment (JornShen) [reply content/en/docs/tasks/tools/_index.md]: ok, learn it :)

File: content/en/docs/tasks/tools/_index.md [hunk 4]
```diff
@@ -47,3 +47,13 @@ installed and configured.
 you need to do to get up and running with `kind`.
 
 <a class="btn btn-primary" href="https://kind.sigs.k8s.io/docs/user/quick-start/" role="button" aria-label="View kind Quick Start Guide">View kind Quick Start Guide</a>
+
+## kubeadm
+
+You can use the {{< glossary_tooltip term_id="kubeadm" text="kubeadm" >}} tool to create and manage Kubernetes clusters.
+It performs the actions necessary to get a minimum viable, secure cluster up and running in a user friendly way.
+
+[`Quick Install`](/docs/setup/production-environment/tools/kubeadm/install-kubeadm/) show you how to install `kubeadm`.
```
Comment (sftim) [inline content/en/docs/tasks/tools/_index.md:56]: I checked the title of the page this links to. I think it should be:
```suggestion
[Installing kubeadm](/docs/setup/production-environment/tools/kubeadm/install-kubeadm/) show you how to install `kubeadm`.
```

**Review by sftim — COMMENTED** [review]
/lgtm

(OK to merge even with the nit I spotted not yet fixed)

File: content/en/docs/tasks/tools/_index.md (hunk 4 above)
Comment (neolit123) [reply content/en/docs/tasks/tools/_index.md:56]: please also change to:
> shows you how to install kubeadm.

- ^ no quotes around "kubeadm"
- show -> shows

Comment (JornShen) [reply content/en/docs/tasks/tools/_index.md]: ok

**Review by sftim — COMMENTED** [review]
/lgtm

**Review by neolit123 — COMMENTED** [review]
/lgtm

**Pull request merged**